src/main/java/com/google/api/codegen/transformer/py/PythonSampleOutputImportTransformer.java

```diff
@@ -60,21 +61,24 @@ private static void addEnumImports(
       ImmutableSet.Builder<ImportFileView> imports,
       MethodContext context,
       OutputView.PrintView view) {
-    for (OutputView.VariableView variableView : view.args()) {
-      TypeModel type = variableView.type();
-      if (type != null && type.isEnum()) {
-        ImportTypeView importTypeView =
-            ImportTypeView.newBuilder()
-                .fullName("enums")
-                .type(ImportType.SimpleImport)
-                .nickname("")
-                .build();
-        imports.add(
-            ImportFileView.newBuilder()
-                .moduleName(context.getNamer().getVersionedDirectoryNamespace())
-                .types(Collections.singletonList(importTypeView))
-                .build());
-      }
+    for (PrintArgView arg : view.args()) {
+      for (PrintArgView.ArgSegmentView segment : arg.segments())
+        if (segment instanceof PrintArgView.VariableSegmentView) {
```

Comment:
FYI: I think stream can be appropriate here. Something like this might work.

```java
boolean addEnumImport = view
  .args()
  .stream()
  .flatMap(arg -> arg.segments().stream())
  .filter(seg -> seg instanceof PrintArgView.VariableSegmentView)
  .map(seg -> ((PrintArgView.VariableSegmentView) seg).variable().type())
  .anyMatch(type -> type != null && type.isEnum());
if (addEnumImport) {
  imports.add(...)
}
```